tests/core/pyspec/eth2spec/test/deneb/transition/test_operations.py

```diff
@@ -83,9 +83,12 @@ def test_transition_attestation_from_previous_fork_with_new_range(
         # NOTE: attestation format changes from Deneb to Electra
         # so the attestation must be made with the `post_spec`
         target_spec = post_spec
+        target_state = post_spec.upgrade_to_electra(state.copy())
+        target_state.fork = state.fork
```

Comment:
this is a bit of a hack, will clean up once we know how we are handling this

Reply:
made a note here to clean this up once we have decided how to handle: https://github.com/ethereum/consensus-specs/issues/3664